notebooks/llm-agent/llm-agent.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        !optimum-cli export openvino --model {model_id} --weight-format int4 {model_path}
</pre><p>need to add $ before {} for resolving variables in cli</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
done